docs/source/i523/2017/organization.rst

```diff
@@ -75,6 +81,8 @@ EOrganization.5:
              the end of Tuesday, meaning you get feedback back on
              Wednesday. Thus dependent on when you ask it may take 4
              days for an answer.
+=======
+>>>>>>> Spelling correction
```

Comment:
This >>>>> is an error and needs to be removed. I guess you have copied it from the command line terminal by mistake.